mk/mockery.mk

```diff
@@ -8,4 +8,6 @@ $(GO_OUTPUT)/mockery: ## Install mockery locally on your GO_OUTPUT (./release) d
 
 .PHONY: mock
 mock: $(GO_OUTPUT)/mockery ## Install mockery if it isn't already in ./release directory and regenerate mocks
-	go generate ./...
\ No newline at end of file
+	sed -i '2,$$ s/^/\/\//' ./pkg/dao/registry_mock.go
```

Comment:
i see this error running `make mock` on a mac:

`sed: 1: "./pkg/dao/registry_mock.go": invalid command code .`

i think it's because the sed tools for GNU and BSD work slightly differently. the commands you have do work on fedora. maybe we can conditionally run one or the other depending on the user's OS? or maybe there's a command that works cross-platform? 

looks like this works on macs:
```
sed -i '' '2,$$ s/^/\/\//' ./pkg/dao/registry_mock.go
sed -i '' '2,$$ s/^\/\///' ./pkg/dao/registry_mock.go
```

Reply:
hrm, didn't think sed would require behave differently between GNU and BSD 🤔
pushed a fix for this, so please try again 😄

Reply:
yep it works now! 🎉